core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4647,6 +4648,37 @@ paths:
                 $ref: '#/components/schemas/ErrorReport'
         500:
           $ref: '#/components/responses/RawlsInternalError'
+  /api/googleProjects:
+    post:
+      tags:
+        - googleProjects
+      summary: register google project in rawls
+      description: Adds a link between a billing project and a google project
+      operationId: createGoogleProject
+      requestBody:
+        description: create google project request
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/CreateGoogleProjectRequest'
+        required: true
+      responses:
+        201:
+          description: Successfully added Google Project in Rawls
+        403:
+          description: lacking permission for billing and/or google project
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
+        409:
+          description: google project is already linked to a billing project
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ErrorReport'
```

Comment:
Along the same lines as [this comment](https://github.com/broadinstitute/rawls/pull/3217/files#r1993784544), in order to allow for idempotent stairway steps, I don't think we want to throw this error. We definitely wouldn't want to throw it if the google project is already linked to the correct billing project. But, I also think it's possible that we just want to be able to override any existing links.

Reply:
Ok sounds like it should be a PUT then?  And just update any existing rows to match the input?  I know there was chatter about that in the ticket I just wasn't sure what the conclusion was.

Reply:
Yeah, I think this lends itself to be a PUT. Perhaps @pshapiro4broad  can speak to this a bit more!

Reply:
Ok I've changed it to a PUT - should I always return a 200 on success?  A 201?  Should I specify 304 if nothing was modified?

Reply:
I think a 200 would be fine because I don't think we want to handle the case that nothing was modified in a different manner.